### PR TITLE
Implement a CockroachDB StatefulSet Use Case

### LINKED ISSUE #851: Update CockroachDB README document with deployment and usage steps
## Is this a BUG REPORT or FEATURE REQUEST?
FEATURE REQUEST

The YAML files for CockroachDB statefulset are available here  : https://github.com/openebs/openebs/tree/master/k8s/demo/cockroachDB

The respective README file has to be updated with relevant details such as : 

- How to deploy
- How to verify successful deployment
- Inserting and retrieving data
- Viewing stats etc.,

### PR DESCRIPTION
**Pull request by yudaykiran**
**What this PR does / why we need it**:
- Create ReadME.md consisting of steps for deployment and use-case demonstration.
- Add a Job YAML for generating load into the CockroachDB database.
- Update the StorageClass YAML to adhere to the new changes made to the storage policies.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #851 

**Special notes for your reviewer**:
Signed-off-by: yudaykiran <udaykiran.y@gmail.com>

### CODE REVIEW

File: k8s/demo/cockroachDB/README.md [hunk 1]
```diff
@@ -1,5 +1,219 @@
...
+## Deploy as a StatefulSet
+
+Deploying CockroachDB as a StatefulSet provides the following benefits:
+
+- Stable unique network identifiers.
+- Stable persistent storage.
+- Ordered graceful deployment and scaling.
+- Ordered graceful deletion and termination.
+
+## Deploy CockroachDB with Persistent Storage
```
Comment (mahebbar) [inline k8s/demo/cockroachDB/README.md]: Deploy = Deploying

File: k8s/demo/cockroachDB/README.md [hunk 2]
```diff
@@ -1,5 +1,219 @@
...
+
+Before getting started check the status of the cluster:
```
Comment (mahebbar) [inline k8s/demo/cockroachDB/README.md]: Before getting started = Before starting, check

File: k8s/demo/cockroachDB/README.md [hunk 3]
```diff
@@ -1,5 +1,219 @@
...
+```bash
+ubuntu@kubemaster:~kubectl get nodes
+NAME            STATUS    AGE       VERSION
+kubemaster      Ready     3d        v1.8.2
+kubeminion-01   Ready     3d        v1.8.2
+kubeminion-02   Ready     3d        v1.8.2
+
+```
+
+Download and Apply the CockroachDB YAMLs from OpenEBS repository:
```
Comment (mahebbar) [inline k8s/demo/cockroachDB/README.md]: Apply = apply

File: k8s/demo/cockroachDB/README.md [hunk 4]
```diff
@@ -1,5 +1,219 @@
...
+Get the status of running StatefulSet:
+
+```bash
+ubuntu@kubemaster:~$ kubectl get statefulset
+NAME          DESIRED   CURRENT   AGE
+cockroachdb   3         3         22h
+
+```
+
+Get the status of underlying persistent volume being used by CockroachDB StatefulSet:
```
Comment (mahebbar) [inline k8s/demo/cockroachDB/README.md]: being used = used

File: k8s/demo/cockroachDB/README.md [hunk 5]
```diff
@@ -1,5 +1,219 @@
...
+
+```
+
+3. Exit the SQL shell:
+
+```sql
+>\q
+```
+
+### Use a Load Generator
```
Comment (mahebbar) [inline k8s/demo/cockroachDB/README.md]: Use = Using

File: k8s/demo/cockroachDB/README.md [hunk 6]
```diff
@@ -1,5 +1,219 @@
...
+
+1. Download and Apply the CockroachDB load generator from OpenEBS repository:
```
Comment (mahebbar) [inline k8s/demo/cockroachDB/README.md]: from OpenEBS = from the OpenEBS

File: k8s/demo/cockroachDB/README.md (hunk 3 above)
Comment (mahebbar) [inline k8s/demo/cockroachDB/README.md]: from OpenEBS = from the OpenEBS

File: k8s/demo/cockroachDB/README.md [hunk 7]
```diff
@@ -1,5 +1,219 @@
...
+cockroachdb-public                                  ClusterIP   10.98.208.2      <none>        26257/TCP,8080/TCP   22h
+kubernetes                                          ClusterIP   10.96.0.1        <none>        443/TCP              20d
+maya-apiserver-service                              ClusterIP   10.98.148.4      <none>        5656/TCP             22h
+pvc-42e9cafc-d4d7-11e7-8d7b-000c29119159-ctrl-svc   ClusterIP   10.96.109.197    <none>        3260/TCP,9501/TCP    22h
+pvc-7005a715-d4d7-11e7-8d7b-000c29119159-ctrl-svc   ClusterIP   10.105.222.30    <none>        3260/TCP,9501/TCP    22h
+pvc-ef78ba18-d4d6-11e7-8d7b-000c29119159-ctrl-svc   ClusterIP   10.110.107.240   <none>        3260/TCP,9501/TCP    22h
+
+```
+
+## Test your Database
```
Comment (mahebbar) [inline k8s/demo/cockroachDB/README.md]: Test your = Testing your

Comment (mahebbar) [reply k8s/demo/cockroachDB/README.md]: No content or is it starting with Use the built-on SQL Client?

File: k8s/demo/cockroachDB/README.md [hunk 8]
```diff
@@ -1,5 +1,219 @@
...
+
+### Use the built-in SQL Client
```
Comment (mahebbar) [inline k8s/demo/cockroachDB/README.md]: Use the = Using the

**Comment by yudaykiran** [comment]
@mahebbar , review comments implemented.

**Pull request merged**